docs/features.rst

```diff
@@ -772,3 +772,54 @@ will not show you exception tracebacks because they are seldom
 helpful in that case.
 
 CumulusCI also has a `--debug` command line argument that may help you investigate bugs.
+
+
+Creating an Unlocked Package
+=============================
+
+While CumulusCI was originally created with a focus on developing managed packages,
+it can also be used to develop and release `unlocked packages <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_intro.htm>`_.
+
+Prerequisites
+-------------
+
+In order to create unlocked package versions, you need to have a few things set up:
+
+1. `Enable Dev Hub in Your Org <https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_enable_devhub.htm>`_
+2. `Enable Unlocked and Second-Generation Managed Packaging <https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_enable_secondgen_pkg.htm>`_
+3. Connect the Dev Hub org to the CumulusCI keychain by running ``cci org connect devhub`` (this is necessary even if sfdx has already authenticated to the Dev Hub).
+4. If you want to create an unlocked package with a namespace, you must also create a new Developer Edition org to `Create and Register Your Namespace <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_create_namespace.htm>`_, and link the namespace to your Dev Hub.
+
+Create a package version
+------------------------
+
+To create a new unlocked package version, run the ``create_package_version`` task against the Dev Hub org:
+
+.. code-block:: console
+
+    $ cci task run create_package_version --org devhub -o package_type Unlocked
+
+This task will look for an unlocked package with the name and namespace specified in the task options (defaulting to the name and namespace from the ``project__package`` section of ``cumulusci.yml``). If a matching package doesn't exist yet, it will be created. Then the task will start creating a new version of this package.
+
+Once creation of the package version has completed (which can take some time), the task will output some information including the SubscriberPackageVersion Id, which can be used to install the package in another org.
```

Comment:
```suggestion
This task will look for an unlocked package with the name and namespace specified in the task options (defaulting to the name and namespace from the ``project__package`` section of ``cumulusci.yml``). If a matching package doesn't exist yet, it will be created. Then the task will start creating a new version of this package.

The task submits a request to create the package version, and once completed (which can take some time), the task will output some information including the SubscriberPackageVersion Id, which can be used to install the package in another org.
```